gradle/verification-metadata.xml

```diff
@@ -2265,6 +2265,11 @@
             <sha256 value="eabaa0ba7428117e8c5cd0188c76d2cc8c92e9b331ee0aee83f815fe8f4d76fa"/>
          </artifact>
       </component>
+      <component group="org.gradle.kotlin" name="plugins" version="1.3.5">
```

Comment:
:x: verification metadata for 1.3.3 shouldn't be needed anymore, let's remove it

Reply:
Done.